src/main/java/de/lukweb/justmail/smtp/SmtpResponse.java

```diff
@@ -20,7 +20,7 @@
     ARGUMENT_ERROR(501, "Synatx error in parameters or arguments"),
     COMMAND_NOT_IMPLEMENTED(502, "SmtpCommand not implemented"),
     BAD_SEQUENCE(503, "Bad sequence of commands"),
-    ARGUMENT_NOT_IMPLEMENTED(504, "SmtpCommand parameter not implemented"),
```

Comment:
I wouldn't change this response according to the [RFC 5321](https://tools.ietf.org/html/rfc5321#section-4.2.2)